src/Compositor.cpp

```diff
@@ -1109,6 +1109,21 @@ wlr_surface* CCompositor::vectorToLayerSurface(const Vector2D& pos, std::vector<
     return nullptr;
 }
 
+SIMEPopup* CCompositor::vectorToIMEPopup(const Vector2D& pos, std::list<SIMEPopup>& popups) {
+    for (auto& popup : popups) {
+        auto surface = popup.pSurface->surface;
+        CBox box{
```

Comment:
did clang-format allow this? or did you forget to run it?

Reply:
This was formatted by clang-format.

Reply:
lol ok thats odd.